nsot/models.py

```diff
@@ -28,7 +28,7 @@
 # These are constants that becuase they are tied directly to the underlying
 # objects are explicitly NOT USER CONFIGURABLE.
 RESOURCE_BY_IDX = (
-    'Site', 'Network', 'Attribute', 'Device', 'Interface'
+    'Site', 'Network', 'Attribute', 'Device', 'Interface', 'Circuit'
```

Comment:
Wondering as we grow the base set of resources if there might not be a better way to do this? It's obviously a not a huge deal, but now we've already got `RESOURCE_BY_IDX`, `VALID_ATTRIBUTE_RESOURCES` (which is actually redundant w/ the class names of `Resource` subclasses), and in #231 we've introduced `HOOK_MODELS` that is pretty much also a direct dupe of `RESOURCE_BY_IDX`.